ext/snmp/snmp.c

```diff
@@ -1827,48 +1830,46 @@ PHP_SNMP_LONG_PROPERTY_READER_FUNCTION(exceptions_enabled)
 /* {{{ */
 static int php_snmp_write_info(php_snmp_object *snmp_object, zval *newval)
 {
-	php_error_docref(NULL, E_WARNING, "info property is read-only");
+	zend_throw_error(NULL, "info property is read-only");
```

Comment:
Just an idea: we could also use the `Read-only property SNMP::$info cannot be written` (?) or a similar construct. Although, `SNMP` should be changed to the active class in this case, since it is not final.

Reply:
That seems like a good format, but don't e have other read only properties in php-src? Maybe better to align it that if they exist.

Reply:
We do have some, but all use their custom format. I plan to polish these in the future. Of course, you don't have to change this message now, we can do it later, separately.

Reply:
I'd rather let you handle that then :)

Reply:
We should probably at least make this
```suggestion
	zend_throw_error(NULL, "SNMP::$info property is read-only");
```
for now